### ISSUE TITLE
Every 5 min EntityConnection.Open is blocked on wait time

### ISSUE THREAD

**Issue by FrancoisM** [aspnet/EntityFramework6#209]
I profiled and did some test on the exact same request.
I'm the only one querying it.
The request is very simple.
If there is more than 5 minutes between calls, the request is very slow and profiling shows that EntityConnection.Open is awaiting (for roughly 3s). Otherwise the request is executed normally.
More details on [SOF](http://stackoverflow.com/questions/42422941/entity-framework-6-blocked-time-waiting-for-a-connection).

**Comment by FrancoisM** [comment]
I think it is related to Metadata caching although I don't understand it yet.

**Comment by rowanmiller** [comment]
What version of EF are you using? There was some timer code in old versions, but we think we removed some of this in later versions.

**Comment by FrancoisM** [comment]
I'm on 6.1.3.
But I have the same issue with waiting a SqlConnection so finally I don't think it is EF related.

**Comment by rowanmiller** [comment]
SqlClient is maintained in https://github.com/dotnet/corefx, can you open a new issue there.
